### PR TITLE
set invalid colour to blue

### PR DESCRIPTION
**Pull request by MagdaN**
removes red border of all required form fields

### CODE REVIEW

**Comment by kleingeist** [comment]
why did you keep the :invalid block? what happens if we just remove the whole block?
now we are overwriting the box-shadow property for `invalid` input elements. is this intendend?

**Comment by MagdaN** [comment]
just removing the block keeps it red

**Comment by kleingeist** [comment]
ah i see the problem
we are still including the form.scss from meinberlin
i'd propose to copy the whole file to beteiligung.in then we don'T have to overwrite

**Comment by MagdaN** [comment]
I agree, but can we make an issue out of it?

**Comment by kleingeist** [comment]
yes. thats fine with me. i'll write it and merge then.
all hail to the deadline ;)

File: liqd_product/assets/scss/_form.scss [hunk 1]
```diff
@@ -4,9 +4,9 @@ textarea {
     box-shadow: 0 0 2px $brand-primary-tint inset;
 
     &:invalid {
-        border-color: $brand-danger;
-        box-shadow: 0 0 2px $brand-danger-tint inset;
-        color: $brand-danger;
+        border-color: $brand-primary;
+        box-shadow: 0 0 2px $brand-primary-tint inset;
+        color: $brand-primary;
```
Comment (kleingeist) [inline liqd_product/assets/scss/_form.scss]: last note: in the base _form.scss the color is set to `contrast-color($body-bg);`  [due to the input property `background-color: $body-bg;`]

for uniformity we should use that, too

**Pull request merged**